distribution/src/site/content/configuration.html

```diff
@@ -315,10 +315,6 @@ <h3>Scanning for steps</h3>
 
 <h3>Configurable Elements</h3>
 
-<p>All configurable elements of <a
-    href="javadoc/core/org/jbehave/core/Configuration.html">Configuration</a>
-come with a default behaviour, configured in the <a
-    href="javadoc/core/org/jbehave/core/MostUsefulConfiguration.html">MostUsefulConfiguration</a>:</p>
```

Comment:
let's do not mix up changes, despite the fact this statement should be updated, the changes are not about it